scripts/generate_env.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+CONFIG_FILE="env/openpaas.js"
+
+mkdir -p env
+
+# Recreate config file
+rm -rf ./$CONFIG_FILE
+touch ./$CONFIG_FILE
+
+# Add assignment
+echo "window.openpaas = {" >> ./$CONFIG_FILE
```

Comment:
Pretty sure you can just `sed` the file without having to read it line by line.
I am not a bash expert at all but look, you can get all the env variables starting with `OPENPAAS`

```sh
bash-3.2$ export OPENPAAS_BASE_URL="http://localhost:8080"
bash-3.2$ env | grep "OPENPAAS_.*"
OPENPAAS_BASE_URL=http://localhost:8080
bash-3.2$ env | awk -F "=" '{print $1}' | grep "OPENPAAS.*"
OPENPAAS_BASE_URL
bash-3.2$ 
```

Then you can iterate on it and sed the json file.

Reply:
Thank you. Please check the new code

Reply:
@tuanlc : because you are seeing the updated code.
In the previous code, I relied on a .env file. This script will read that .env file line by line to get the environment variables. 
Christophe suggested using grep and I updated it.

Reply:
Ok